OpenRA.Mods.RA/Traits/Infiltration/Infiltrates.cs

```diff
@@ -58,6 +58,10 @@ bool IsValidOrder(Actor self, Order order)
 			if (order.ExtraData == 0 && order.TargetActor == null)
 				return false;
 
+			// Should not infiltrate own or allied
+			if (order.TargetActor.Owner.IsAlliedWith(self.Owner))
```

Comment:
Not certain what you mean - that sounds pointless/abusable to me. Issuing orders before breaking a fragile alliance for infiltration would be the sole case I'd see that field being used and I consider that an abuse.